python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -246,6 +265,11 @@ def _update(self):
                 else:
                     logger.error(f"StandardAutoscaler: {node_id}: Terminating "
                                  "failed to setup/initialize node.")
+                    self.event_summarizer.add(
+                        "Removing {} nodes of type " +
+                        self._get_node_type(node_id) + " (setup failed).",
```

Comment:
```suggestion
                        self._get_node_type(node_id) + " (launch failed).",
```
setup failed can be misleading, it can be setup/initialization/docker/anything the command runner does.